### DISCUSSION TITLE
Is there a way to disable WYSIWYG keyboard shortcuts in the Editor?

### DISCUSSION THREAD

**Discussion by paridhikhaitan** [ueberdosis/tiptap#786] [General Questions]
I have a functionality wherein I want to disable keyboard shortcuts (specifically Ctrl+B, Ctrl+I and Ctrl+U). Does Tiptap have this option?

**Comment by hanspagel** [comment]
One possible solution is to copy the default marks and remove the keys, e. g. for bold:
https://github.com/ueberdosis/tiptap/blob/e3058f7cfadc33ab2f213ed5cceab9c6e3ece257/packages/tiptap-extensions/src/marks/Bold.js#L29-L33

Another solution would be to overwrite the keymaps, like mentioned in #207.

Hope that helps!
